yesod-bin/AddHandler.hs

```diff
@@ -247,4 +250,8 @@ getSrcDir cabal = do
 #endif
     let buildInfo = allBuildInfo pd
         srcDirs = concatMap hsSourceDirs buildInfo
-    return $ fromMaybe "." $ listToMaybe srcDirs
+#if MIN_VERSION_Cabal(3, 6, 0)
+    return $ maybe "." getSymbolicPath $ listToMaybe srcDirs
+#else
+    return $ fromMaybe "." $ listToMaybe srcDirsFilePath
```

Comment:
I'm not a regular haskell code reviewer, so sorry if I'm wrong. But where does `srcDirsFilePath` come from? I can't find the definition in this file.

Reply:
Good spot! That was a typo the "FilePath" shouldn't be there. I've force pushed a fix